packer/resources/features/logstash/install.sh

```diff
@@ -18,9 +18,22 @@ ln -s /opt/features/logstash/logstash /opt/logstash
 # TODO could use the handy substitution script once #23 is merged
 ESCAPED_USER=$(echo $USER | sed -e 's/[\/&]/\\&/g')
 ESCAPED_CONFIGFILE=$(echo $CONFIGFILE | sed -e 's/[\/&]/\\&/g')
+
+# Upstart
+if [ -d "/etc/init" ]; then
```

Comment:
I'm trying to make the features work on both the Trusty and Vivid images. But you're right, it might be pretty confusing to have the service installed in 2 different ways on Vivid.

I'll change the logic to "if there's a systemd directory, install a systemd script, otherwise install an upstart script".